lib/cluster/rds.rb

```diff
@@ -199,5 +213,25 @@ def self.get_parameters
       }
     end
 
+    def self.get_update_parameters
+      parameters = get_parameters
+      parameters.delete(:timeout_in_minutes)
+      parameters
+    end
+
+    def self.create_change_set(parameters)
+      parameters[:change_set_name] = "rds-update-#{Time.now.to_i}"
+      resp = cloudformation_client.create_change_set(parameters)
+      change_set = cloudformation_client.describe_change_set({
+        change_set_name: resp.id
+      })
+      if change_set.status == 'FAILED' && change_set.status_reason =~/didn't contain changes/
+        cloudformation_client.delete_change_set({
+          change_set_name: resp.id
+        })
+        raise "No updates"
+      end
+      puts "Change set #{change_set.change_set_name} created. Review and approve via Cloudformation web console."
+    end
```

Comment:
moved creation of cloudformation change set and creation params into class methods